cmds/truncate/truncate.go

```diff
@@ -46,17 +50,11 @@ func usageAndExit() {
 func main() {
 	flag.Parse()
 
-	if flag.NArg() == 0 {
+	if flag.NArg() == 0 || !size.IsSet {
```

Comment:
I'd prefer to have this broken out into two ifs which are clear about the error. Maybe it's just me, but I go crazy with error messages of the form "one of these things is wrong, guess which one", especially if it's from deep within a script.

Reply:
OK